test/conformance/tests/sriov_operator.go

```diff
@@ -1216,6 +1216,100 @@ var _ = Describe("[sriov] operator", func() {
 			})
 		})
 
+		Context("Nic Validation", func() {
+			numVfs := 5
+			resourceName := "testresource"
+
+			BeforeEach(func() {
+				if discovery.Enabled() {
+					Skip("Test unsuitable to be run in discovery mode")
+				}
+			})
+
+			findSriovDevice := func(vendorID, deviceID string) (string, sriovv1.InterfaceExt) {
+				for _, node := range sriovInfos.Nodes {
+					for _, nic := range sriovInfos.States[node].Status.Interfaces {
+						if vendorID != "" && deviceID != "" && nic.Vendor == vendorID && nic.DeviceID == deviceID {
+							return node, nic
+						}
+					}
+				}
+
+				return "", sriovv1.InterfaceExt{}
+			}
+
+			DescribeTable("Test connectivity using the requested nic", func(vendorID, deviceID string) {
+				By("searching for the requested network card")
+				node, nic := findSriovDevice(vendorID, deviceID)
+				if node == "" {
+					Skip(fmt.Sprintf("skip nic validate as wasn't able to find a nic with vendorID %s and deviceID %s", vendorID, deviceID))
+				}
+
+				By("creating a network policy")
```

Comment:
Don't you need to clean / wait for stable here?

Reply:
I have the wait in line 1284 https://github.com/openshift/sriov-network-operator/pull/306/files#diff-96912b0a2687700a01b25f6cdad2659fR1284

and the clean is part of the before each outside this context

Reply:
Cool, thanks!